### DISCUSSION TITLE
What should we rename our organization to?

### DISCUSSION THREAD

**Discussion by cosmiccoincidence** [RE-SS3D/SS3D#728] [General]
It has been suggested that we rename our organization to something more interesting as currently "SS3D being developed by RE-SS3D" doesn't sound very creative.

This will eventually be used for more than just our github organization name, such as when launching the game it may say "Space Station 3D" but below it may also say "Developed by [insert our organization name]".

An example of what I mean can be seen in the other 2 primary SS13 remakes:
- UnityStation being developed by "UnityStation"
- SpaceStation14 being developed by "Space Wizards Federation"

The latter of which is more creative and unique. The former is boring and possibly even confusing.

Post your ideas for our new organization name below and vote on your favorites.

**Comment by AmplifierSoftware** [comment]
soyjack softworks

**Comment by cosmiccoincidence** [comment]
Honk Productions

**Comment by cosmiccoincidence** [comment]
*beep
*beep Studios
*beep Productions
*beep Creations
BeepStation
beeple

**Comment by cosmiccoincidence** [comment]
Schrodinger's Bus

**Comment by cosmiccoincidence** [comment]
Space Monkeys

**Comment by cosmiccoincidence** [comment]
Floor Pill Adventures

**Reply by stilnat** [reply]
I love this one

**Comment by Dropgunner** [comment]
Allied Mastercomputing

**Comment by joaoburatto** [comment]
Spessman Incorporated

**Comment by AmplifierSoftware** [comment]
chungus inc

**Comment by AmplifierSoftware** [comment]
sneeple

**Comment by AmplifierSoftware** [comment]
amogo 13

**Comment by cosmiccoincidence** [comment]
**Others from discord:**
Incompetent Crew
Drunken Spessman
Bald Captain
Bad Clown

**Reply by Birlinha** [reply]
Clown Station 13

**Reply by Birlinha** [reply]
Clown Captain 13
CC13

**Reply by Birlinha** [reply]
I mean - CC13 looks like an amazing logo. Gimme a minute

**Reply by Birlinha** [reply]
![Yes, I fucking overdid it](https://user-images.githubusercontent.com/47873385/132959584-6794669d-e361-4730-b1ff-b963055bdc11.jpg)

**Comment by Birlinha** [comment]
Why not use SS13 lore for the name?

Nanotrasen Space Programming
Nanotrasen Game Engineering
Nanotrasen Space Simulations

**Reply by Birlinha** [reply]
Nanotrasen VR

**Reply by Birlinha** [reply]
NVSS

Nanotrasen Virtual Space Station

**Reply by Birlinha** [reply]
NSS13
Nanotrasen Space Station 13
or
NS13
Nanotrasen Station 13

I notice they don't mention it being 3D, but maybe someone will connect the dots

**Reply by joaoburatto** [reply]
Nanotrasen Engineering is a good one, or Nanotrasen Incorporated

**Comment by Dennis-Nesterenko** [comment]
Centcom Softworks, can make the abbreviation fancy like C·S or something

- Short and simple
- Could easily make a good logo based off the name
- I don't believe it's trademarked or anything
- We already consider ourselves Centcom so might as well
- Rolls off the tounge nicely 
- Afaik we can't really use names like Nanotransen as it's already under some form of copyright (?)
- Could go kooky with it and instead call it Centcom Softwerks or something lmao

**Comment by Liam-Harrison** [comment]
I would like CENTCOM Softworks, however I think CENTCOM is fine too.

**Comment by cosmiccoincidence** [comment]
Desperate Space Mimes

**Comment by cosmiccoincidence** [comment]
### Extra-Dimensional

I like this idea because we (and/or others) might 'port' other 2 dimensionsal games to 3D using SS3D as a base, giving them "an extra dimension". They would be another server on our hub of course. Some potential ideas:
- Aliens 3D - basically the CM ss13 server, which took the alien gamemode from ss13 and expanded on it heavily
- Infected 3D - some ss13 servers already have zombies and zombie outbreaks, now expand on it heavily with a lot of influence from CDDA and Project Zomboid. (I've wanted to do this for a while)
- Rimworld 3D (needs better name) - kinda like some of the mini gamemodes of ss13 which include salvaging, mining, and base building... now expand on that and you basically have Rimworld multiplayer 3D.
- Dwarf Fortress 3D (needs better name) - same thing as above really, just different setting.
- Fallout Something (needs better name) - it's been done many times over in ss13, you know damn well someone will do it in 3d. Just needs to avoid erp and it will succeed.
- inb4 someone "heavily expand" lightsaber dueling and we have a star wars server. 

I concede that this name could use an 'ending', something like (open to suggestions):
- Extra-Dimensional Creations
- Extra-Dimensional Games
- Extra-Dimensional Gaming
- Extra-Dimensional Gamers
- Extra-Dimensional Productions

**Reply by cosmiccoincidence** [reply]
Kirby likes "Extradimensional Productions"

**Comment by iamteapot422** [comment]
Nanotransen Engineering

**Comment by Dennis-Nesterenko** [comment]
Nanotrasen Department of Software Engineering

**Comment by cosmiccoincidence** [comment]
Nanotrasen Research Productions (idea by Kirby)

**Comment by Ratvarr** [comment]
The syndicate

**Comment by MaxIsJoe** [comment]
3rd Spessmen

**Comment by Fogapod** [comment]
unitystation

**Comment by yrokhlin** [comment]
Tesloose

**Comment by Em3rgencyLT** [comment]
Singularity

**Comment by cosmiccoincidence** [comment]
Spaced & FloorPilled

**Comment by CapoFantasma97** [comment]
Out Of The Toolbox

**Comment by CapoFantasma97** [comment]
~~Toolbox Gaming~~
